tools/cli/admin.go

```diff
@@ -767,108 +771,53 @@ func newAdminQueueCommands() []cli.Command {
 }
 
 func newDBCommands() []cli.Command {
+	var collections cli.StringSlice = invariant.CollectionStrings()
+
+	scanFlag := cli.StringFlag{
+		Name:     FlagScanType,
+		Usage:    "Scan type to use: " + strings.Join(executions.ScanTypeStrings(), ", "),
+		Required: true,
+	}
+
+	collectionsFlag := cli.StringSliceFlag{
+		Name:  FlagInvariantCollection,
+		Usage: "Scan collection type to use: " + strings.Join(collections, ", "),
```

Comment:
Maybe add some example collection type here?

Reply:
All available collections will be printed out here, like this:
```
  --invariant_collection value    Scan collection type to use: CollectionMutableState, CollectionHistory (default: "CollectionMutableState", "CollectionHistory")
```